WordPress/wordpress-playground
# Fix progress bar regression introduced in #639

PR #639 removed the `export { dependencyFilename };` line from the `php-wasm/web/public/*/php_*.js` files. This prevented reporting the download progress of the PHP WASM binary files. The following error message was logged to the console:

> worker-thread-175494d0.js?wpVersion=6_4&phpVersion=8_0&networking=no&storage=none:52 Registered a download #progress of an unregistered file "php_8_0-c4af17c8.wasm". This may cause a sudden **decrease** in the #progress percentage as the total number of bytes increases during the download.

This PR adds the missing line back to the files.

 ## Testing instructions

1. Open the playground
2. Confirm there is no error message in the console
3. Confirm the progress bar is working as expected (this might only be possible to test after the next release as locally the WASM files are downloaded instantly even when throttling is enabled)

cc @seanmorris
